lib/graphql/schema/relay_classic_mutation.rb

```diff
@@ -105,7 +105,7 @@ def field_options
           sig = super
           # Arguments were added at the root, but they should be nested
           sig[:arguments].clear
-          sig[:arguments][:input] = { type: input_type, required: true }
+          sig[:arguments][:input] = { type: input_type, required: true, description: input_type.description }
```

Comment:
It seems like this will generate: 

```
someMutation(
  input: SomeInput # Autogenerated input type of SomeMutation
 )
 ```
 
 But that description is for the _type_, not the argument. How about using an argument-specific description instead? For example:
 
```suggestion
          sig[:arguments][:input] = { type: input_type, required: true, description: "Parameters for #{graphql_name}" }
```